src/ruleset.xml

```diff
@@ -4,7 +4,19 @@
 
     <rule ref="PSR2" />
 
+    <!-- Checks that the opening PHP tag is the first content in a file -->
```

Comment:
@willdoran this adds some of what we discussed today for the new code review guidelines. We shouldn't merge until there's agreement on the rules, just sending this PR so you have it as reference